File: solutions/1833-find-the-highest-altitude/solution.java
class Solution {
    public int largestAltitude(int[] gain) {
        int ans=0,sum=0,n=0;
        for(int i=0;i<gain.length;i++)
        {
            sum=ans+gain[i]+sum;
            if(n<sum)
                n=sum;
        }
        return n;
    }
}

File: solutions/1833-find-the-highest-altitude/solution.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        ans=0
        sum=0
        n=0;
        for i in range(len(gain)):
            sum=ans+gain[i]+sum;
            if(n<sum):
                n=sum;
        return n;
